jax/_src/interpreters/partial_eval.py

```diff
@@ -2382,6 +2383,38 @@ def _jvp_jaxpr_zeros(f, store, in_zeros, zero_avals, *primal_tangent_avals):
   store.store(out_zeros)
   return [*out_primals, *out_nz_tangents]
 
+def trace_to_jaxpr(
```

Comment:
Is there a plan to merge this and the trace_to_jaxpr_dynamic eventually? they are very similar.